### DISCUSSION TITLE
Quais são as extensões mais úteis do VSCode?

### DISCUSSION THREAD

**Discussion by andrewarosario** [techexpertspro/portal-ascepa#32] [Q&A]
Quais extensões do VSCode são indispensáveis para vocês? Aquelas que facilitam e muito o dia-a-dia.
Comentem aqui, que vamos catalogar as mais utilizadas para adicionar no arquivo `.vscode/extensions.json`.

**Comment by guitoledo** [comment]
Intellisense

**Comment by pablodepine** [comment]
Angular Language Service

**Comment by keidsondesigner** [comment]
Angular Language Service
Angular Schematics
Nx Console
Jest Runner
Erro Lens
ESLint
Prettier
Auto import 
Paste JSON as Code
Auto Rename Tag
Auto Close Tag
Sorting HTML
Tailwind CSS IntelliSense
IntelliCode
Path Intelisense
Conventional Commits
Git Graph
Import Cost
Codeium
Move TS

**Comment by Flamilani** [comment]
Gostaria de compartilhar outras extensões:
https://marketplace.visualstudio.com/publishers/GitHub
https://kinsta.com/pt/blog/extensoes-vscode-extensions/
https://warcontent.com/extensoes-vscode-angular/

GitHub Pull Requests
GitHub Actions
Better Comments
Angular Test Snippets
